dashboard/test/controllers/programming_environments_controller_test.rb

```diff
@@ -55,28 +54,9 @@ class ProgrammingEnvironmentsControllerTest < ActionController::TestCase
     assert_response :redirect
   end
 
-  test 'page is proxied to docs show page if not using studio code docs' do
-    Rails.application.config.stubs(:levelbuilder_mode).returns false
-    DCDO.expects(:get).with('use-studio-code-docs', false).returns(false).at_least_once
-
-    programming_environment = create :programming_environment, name: 'weblab'
-    category = create :programming_environment_category, programming_environment: programming_environment
-    create :programming_environment_category, programming_environment: programming_environment
-    create :programming_expression, programming_environment: programming_environment, programming_environment_category: category
-
-    stub_request(:get, "https://curriculum.code.org/docs/#{programming_environment.name}/").
-        to_return(body: 'curriculum.code.org/docs content', headers: {})
-
-    request.host = "studio.code.org"
-    get :docs_show, params: {programming_environment_name: programming_environment.name}
-    assert_response :ok
-    assert_equal @response.body, 'curriculum.code.org/docs content'
-  end
-
   test 'page is not proxied to docs index if using studio code docs' do
     Rails.application.config.stubs(:levelbuilder_mode).returns false
     DCDO.expects(:get).at_least_once
```

Comment:
Not 100% sure but you can likely remove this line too. I think I had to add it because once I started checking DCDO, it was failing because there were `get` calls for unrelated flags